modules/overrides/overrides.go

```diff
@@ -36,13 +38,18 @@ func loadPerTenantOverrides(r io.Reader) (interface{}, error) {
 	return overrides, nil
 }
 
+type Config struct {
+	Defaults           *Limits             `yaml:"defaults"`
+	PerTenantOverrides *perTenantOverrides `yaml:"overrides,omitempty"`
```

Comment:
Not a detailed review, but this seems to be holding information that's present in `tenantLimits` already. Can we club the two somehow? Maybe something like this:
```
type Overrides struct {
	services.Service

	defaultLimits *Limits
	tenantLimits  perTenantOverrides

	// Manager for subservices
	subservices        *services.Manager
	subservicesWatcher *services.FailureWatcher
}
```

Reply:
That's a good point, addressed in 47c4af1cc5a74580d498de08ec8fb26ee3adfffb. Thanks